integration_tests/src/main/python/json_test.py

```diff
@@ -380,6 +380,7 @@ def test_from_json_map_fallback():
         'JsonToStructs',
         conf={"spark.rapids.sql.expression.JsonToStructs": True})
 
+@pytest.mark.xfail(reason='GPU from_json does not support struct type schema')
```

Comment:
once the issue is filed we can use its URL as the reason https://github.com/NVIDIA/spark-rapids/pull/8557#issuecomment-1588246631

Reply:
Changes updated. Thank you!